client/app/index.html

```diff
@@ -246,6 +246,12 @@
 
         config.previewSubjectFilterKey = 'patopic';
 
+        //date formats for publish/embargo entry
+        config.view = {
+            dateformat: 'D/M/YYYY',
```

Comment:
yeah, this time format is for input only in the angular view, the angular model reformats it to server format. The singular just allows them to type 4/4/2016 to save time 9:00 etc...